helm/charts/hpe-csi-driver/Chart.yaml

```diff
@@ -3,10 +3,15 @@ version: "2.2.0"
 appVersion: "2.2.0"
 annotations:
   artifacthub.io/prerelease: "false"
+  artifacthub.io/containsSecurityUpdates: "true"
   artifacthub.io/license: Apache-2.0
   artifacthub.io/links: |
     - name: Documentation
       url: https://scod.hpedev.io/csi_driver
+  artifacthub.io/recommendations: |
+    - url: https://artifacthub.io/packages/olm/community-operators/hpe-csi-operator
+    - url: https://artifacthub.io/packages/helm/hpe-storage/hpe-csi-info-metrics
+    - url: https://artifacthub.io/packages/helm/hpe-storage/hpe-array-exporter
```

Comment:
any reason we would need array exporter package for csi driver chart ?

Reply:
This is just to advertise a "recommendation" on artifacthub. It's not a dependency.